stable/datadog/values.yaml

```diff
@@ -3,7 +3,7 @@ image:
   # This chart is compatible with different images, please choose one
   repository: datadog/agent               # Agent6
   # repository: datadog/dogstatsd         # Standalone DogStatsD6
-  tag: 6.3.2  # Use 6.3.2-jmx to enable jmx fetch collection
+  tag: 6.4.2  # Use 6.4.2-jmx to enable jmx fetch collection
```

Comment:
please update chart appVersion to match tag, update readme to match, and bump the chart version as well.
TIA and thanks for the PR!

Reply:
Just discovered that the Circle build failed, Chart version and appVersion bumped

Reply:
Thanks - see you at JZ18 ;-)